### PR TITLE
Tim Evens updates for issues up to issue 11. 

### LINKED ISSUE #1: Duplication of "filters"
Should "filters" be present twice in this list?

Original:
   Adj-RIB-Out received as Adj-RIB-In from another router may have a policy
   applied that filters, generates aggregates, suppresses more specific
   prefixes, manipulates attributes, or filters routes.

Perhaps:
   Adj-RIB-Out received as Adj-RIB-In from another router may have a policy
   applied that generates aggregates, suppresses more specific prefixes,
   manipulates attributes, or filters routes.

### LINKED ISSUE #3: Loc-RIB route monitoring granularity
For clarity, please consider whether the following update is acceptable.

Original:
   A BMP receiver should expect that Loc-RIB route monitoring granularity can
   be different by BMP sender implementation.

Perhaps:
   A BMP receiver should expect that the granularity of Loc-RIB route monitoring can
   vary depending on the BMP sender implementation.

### LINKED ISSUE #4: IANA updates
We have updated some of the IANA text for clarity and to better align with what appears in the IANA registry.  Please review carefully and let us know if any updates are needed.

### LINKED ISSUE #5: RFC 5226 reference
There seems to be no reference to [RFC5226] in the document.  Please let us know where the citation to [RFC5226] should be placed or whether the reference should be removed.  Perhaps a citation should be added for Standards Action?

### LINKED ISSUE #6: BMP Peer Flags registries
Does the following text refer to the "BMP Peer Flags for 
Peer Types 0 through 2" or "BMP Peer Flags for Loc-RIB Instance Peer Type 3" registry?  In "BMP     
Peer Flags for Loc-RIB Instance Peer Type 3", Flag 0 has been registered as F flag.  In "BMP Peer   
Flags for Peer Types 0 through 2", values 0-3 were assigned in other RFCs, so they're not           
"unassigned".

See the "BGP Monitoring Protocol (BMP) Parameters" registry
<https://www.iana.org/assignments/bmp-parameters>.

Original:
   Flags 0 through 3 and 5 through 7 are unassigned.  The registration
   procedure for the registry is "Standards Action".

### LINKED ISSUE #8: Terminology
Throughout the text, the following terminology appears to be used inconsistently. Please review these occurrences and let us know if/how they may be made consistent.

- Instance Peer type vs. instance peer type vs. Instance peer type

- Route Monitoring message vs. route monitoring message vs. Route Monitor message vs. route
monitor message

### LINKED ISSUE #9: Inclusive language
Please review the ["Inclusive Language"](https://www.rfc-editor.org/styleguide/part2/#inclusive_language) portion of the online Style Guide.  While our script did not flag any of the potentially biased words from [Table 1](https://www.nist.gov/nist-research-library/nist-technical-series-publications-author-instructions#table1) of "NIST Technical Series Publications Author Instructions", please let us know if any changes are needed.

### LINKED ISSUE #10: Example clarity
The "for example" text is a sentence fragment, and it is unclear how it relates to the text that precedes it.  Please clarify.                                           
                                                                                                    
Original:                                                                                           
   It does define in section 8.2 of [RFC7854] locally                                               
   originated routes, but these routes are defined as the routes                                    
   originated into BGP.  For example, as defined by Section 9.4 of                                  
   [RFC4271].

### LINKED ISSUE #11: Setting the Peer BGP ID
We are having trouble parsing this sentence:

Original:
   *  Peer BGP ID: Set to the BGP instance global or RD (e.g., VRF)
      specific router-id section 1.1 of [RFC7854].

a) Does this mean that the BGP instance is set to global or RD depending on the specific router
ID?  Please review.

b) Section 1.1 of RFC 7854 is the "Requirements Language" section - it contains only the RFC 2119
paragraph that defines use of capitalized key words.  Should this text refer to Secion 4.2
instead?

Section 4.2 of RFC 7854 defines Peer BCP ID as follows:

   o  Peer BGP ID: The BGP Identifier of the peer from which the
      encapsulated PDU was received.

Global Instance Peer and RD Instance Peer are also are also described in Section 4.2 of RFC 7854.

### PR DESCRIPTION
**Pull request by TimEvens**
Resolves #3, resolves #4, resolves #5, resolves #6,
resolves #8, resolves #9, resolves #10, resolves #11

### CODE REVIEW

**Review by sginoza — COMMENTED** [review]
#11 Your update looks good - just confirming that we're intentionally getting rid of the reference to 7854 in that sentence.  

#10 I'm ok with this, but I suggested a minor update.

#1 I don't think issue 1 is addressed by this PR; that's ok, I will merge #12 

The rest LGTM!  Thanks for your review and updates.

**Comment by TimEvens** [comment]
> #11 Your update looks good - just confirming that we're intentionally getting rid of the reference to 7854 in that sentence.

Correct. It's not the bgp id of the peer the pdu was received.

> #10 I'm ok with this, but I suggested a minor update.

It has the change you indicated, but I added an eg instead. Are you saying it should be changed?  For example, this change has:

```
 It does define locally originated routes in
   Section 8.2 of [RFC7854], but these routes are defined as the routes
   that originated (e.g., Section 9.4 of [RFC4271]) into BGP.
```

Your suggested change was:

```
 It does define in section 8.2 of [RFC7854] locally
originated routes but these routes are defined as the routes
originated into BGP, for example, as defined by Section 9.4 of
[RFC4271].
```


> #1 I don't think issue 1 is addressed by this PR; that's ok, I will merge #12

That was a mistake to add that it resolves #1.  I left that off intentional so there wouldn't be a merge conflict.

File: rfc9069.xml [hunk 1]
```diff
@@ -86,18 +86,10 @@
...
-   originated routes, but these routes are defined as the routes
-   originated into BGP.  For example, as defined by Section 9.4 of
-   [RFC4271].
--->
 
  It does define locally originated routes in <xref target="RFC7854" sectionFormat="of" section="8.2"/>,
-                but these routes are defined as the routes that originated into BGP. For example, as
-                described by <xref target="RFC4271" sectionFormat="of" section="9.4"/>. Loc-RIB
+                but these routes are defined as the routes that originated (e.g., 
+                <xref target="RFC4271" sectionFormat="of" section="9.4"/>) into BGP. Loc-RIB
```
Comment (TimEvens) [inline rfc9069.xml]: @sginoza , so I should avoid the eg in favor of spelling it out?

**Review by paololucente — COMMENTED** [review]
This looks fine to me. I'd wait this to be merged in order to re-read the whole document to check if any of the term normalization, ie. Peer Type etc., did escape the count. Paolo

File: rfc9069.xml (hunk 1 above)
Comment (sginoza) [reply rfc9069.xml]: @TimEvens  Hi Tim, 
Sorry I wasn't clear.  I  suggest moving "(e.g., Section 9.4 of [RFC4271])" to the end of the sentence for readability.  So, we'd have the following.  If you prefer your update, that's ok - this is minor.

   It does define locally originated routes in
   Section 8.2 of [RFC7854], but these routes are defined as the routes
   that originated into BGP (e.g., Section 9.4 of [RFC4271]).

**Comment by sginoza** [comment]
> I'd wait this to be merged in order to re-read the whole document

@paololucente No problem.  I will merge the PR when it's ready.  Please note that final approvals will still need to be sent via mail to rfc-editor@rfc-editor.org.

**Comment by TimEvens** [comment]
@sginoza , does the latest commit work?

**Comment by sginoza** [comment]
@TimEvens Yes - looks good.  I think we're ready to merge the PR!  I'll take care of it.

**Pull request merged**